Sources/GRPC/AsyncAwaitSupport/GRPCAsyncRequestStreamWriter.swift

```diff
@@ -62,65 +69,17 @@ public struct GRPCAsyncRequestStreamWriter<Request: Sendable>: Sendable {
     _ request: Request,
     compression: Compression = .deferToCallDefault
   ) async throws {
-    try await self.asyncWriter.write((request, compression))
+    try await self.asyncWriter.yield((request, compression))
   }
 
-  /// Finish the request stream for the RPC. This must be called when there are no more requests to
-  /// be sent.
-  ///
-  /// - Throws: ``GRPCAsyncWriterError`` if the request stream has already been finished.
-  public func finish() async throws {
-    try await self.asyncWriter.finish()
+  /// Finish the request stream for the RPC. This must be called when there are no more requests to be sent.
+  public func finish() {
```

Comment:
Oh right, this makes sense. Are we okay with the new `finish(with:)` method to not be `async throws` since it's new? Or should I also mark them as `async throws` for consistency?

Reply:
Oh right, this makes sense. Are we okay with the new `finish(with:)` method to not be `async throws` since it's new? Or should I also mark them as `async throws` for consistency?